test/integration/integration_test.go

```diff
@@ -123,9 +123,9 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`executing "oc ([^"]*)" (succeeds|fails)$`, minishift.executingOcCommandSucceedsOrFails)
 
 	// steps to verify stdout and stderr of commands executed
-	s.Step(`([^"]*) should contain ([^"]*)$`, commandReturnShouldContain)
+	s.Step(`([^"]*) should contain "([^"]*)"$`, commandReturnShouldContain)
```

Comment:
We are doing it to make sure we capture exitcode which is part of `"<exitcode>"` regex so if we have something extra can be part of `"<what_ever>"` part of this. Quite neat 👍